.prettierrc.json

```diff
@@ -1,4 +1,5 @@
 {
   "endOfLine": "auto",
-  "htmlWhitespaceSensitivity": "ignore"
+  "htmlWhitespaceSensitivity": "ignore",
+  "trailingComma": "none"
```

Comment:
My vote is to stay with the Spotless default rather than introduce our own customizations which then need to be maintained.